docs/content/faq.md

```diff
@@ -15,7 +15,7 @@ Yes - you can export your slides in PDF format. Appending your presentation URL
 
 If you want a black & white version of your slides printed to PDF, append your URL with `?export&print` to get a printer-friendly, flattened, black & white print out of your slideshow.
 
-For more info about the query parameters Spectacle supports, please check out our section about it in the [advanced concepts documentation](/docs/advanced-concepts#query-parameters).
+For more info about the query parameters Spectacle supports, please check out our section about it in the [advanced concepts documentation](./advanced-concepts#query-parameters).
```

Comment:
GLOBAL: All docs links should be of the form `/docs/PATH_WITHOUT_TRAILING_SLASH_OR_MD_EXTENSION#HASH`